telegram/ext/stringcommandhandler.py

```diff
@@ -86,18 +97,13 @@ def check_update(self, update):
         return (isinstance(update, string_types) and update.startswith('/')
                 and update[1:].split(' ')[0] == self.command)
```

Comment:
This code is not aligned with the code in `collect_additional_context()`.  I suggest to do:
```python
if isinstance(update, string_types) and update.startswith('/'):
    args = update[1:].split(' ')
    if args[0] == self.command:
        return args
return None
```